Pages/fashionCategoryPage.js

```diff
@@ -0,0 +1,31 @@
+class FashionCategoryPage{
```

Comment:
Same comments as for regression PR.
Is there need to have "FashionCategory" class, where is "Category" class ?
Is there some relation between found page objects you have, inheritance for example?

Reply:
There is no Category page, because Categories are shown on homepage, and every category link opens another category page (for example: Fashion, Shoes, etc...), and every category page contains different data. So, if I would test items from another category page, I would add another class for that category.

Reply:
This will be resolved in regression tests.